### ISSUE TITLE
Can't install the package when using nova local nova installed

### ISSUE THREAD

**Issue by muhghazaliakbar** [ebess/advanced-nova-media-library#171]
I'm having problem with the nova version requirement when installing this package with nova installed using `path` method in composer. https://nova.laravel.com/docs/3.0/installation.html#installing-nova

Anyone can help how to fix it?

**Comment by VeeeneX** [comment]
@muhghazaliakbar Can you please post more details? With log of composer?

**Comment by imjohnbon** [comment]
I'm actually experiencing the same issue with version 3.1. Here is the composer error:

```
  - Installation request for ebess/advanced-nova-media-library ^3.1 -> satisfiable by ebess/advanced-nova-media-library[3.1.0].
- ebess/advanced-nova-media-library 3.1.0 requires laravel/nova ^2.0|^3.0 -> no matching package found.
```

And here is how Nova is referenced in my composer file (per the Nova docs):

```
"laravel/nova": "*",
```

**Comment by imjohnbon** [comment]
Okay @tammytee figured it out, here's what we did to resolve this issue after trying multiple different steps:

1. Replaced our local Nova install in `/nova` with the new version 3 of Nova.
2. Ran `composer remove ebess/advanced-nova-media-library` to remove this package entirely.
3. Deleted our `vendor` folder and our `composer.lock` file.
4. Updated our local `/composer.json` file so that Nova was versioned like this:

```
"laravel/nova": "^3.0"
```

Also added these keys at the bottom of our `composer.json` file (and made sure our Nova repository was setup properly), it looked like this:

```
"minimum-stability": "dev",
"prefer-stable": true,
"repositories": {
    "laravel/nova": {
        "type": "path",
        "url": "./nova"
    }
}
```

5. Updated the `nova/composer.json` file and added `"version": "3.0"` at the top under the "name" key.
6. Ran `composer update`.
7. Checked our new `composer.lock` file and made sure that `laravel/nova` was referenced as `3.0`.
8. Ran `composer require ebess/advanced-nova-media-library`.

After that it finally installed correctly. I hope this helps anyone else who might be having this issue, took a lot of trial and error on our end.
